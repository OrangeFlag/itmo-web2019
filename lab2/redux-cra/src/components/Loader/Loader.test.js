import React from 'react';
import {shallow} from 'enzyme';
import Loader from './Loader.style';

describe('Loader tests', () => {
    it('should render correctly +', () => {
        const container = shallow(<Loader/>);

        expect(container).toMatchSnapshot();
    });
});