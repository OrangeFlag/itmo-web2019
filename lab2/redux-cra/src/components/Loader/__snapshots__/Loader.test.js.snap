// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader tests should render correctly + 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": false,
        "rules": Array [
          "
    border: 16px solid #f3f3f3; /* Light grey */
    border-top: 16px solid #3498db; /* Blue */
    border-radius: 50%;
    width: 120px;
    height: 120px;
    animation: ",
          Keyframes {
            "id": "sc-keyframes-iVXCSc",
            "inject": [Function],
            "name": "iVXCSc",
            "rules": Array [
              "@-webkit-keyframes iVXCSc{from{-webkit-transform:rotate(0deg);-ms-transform:rotate(0deg);transform:rotate(0deg);}to{-webkit-transform:rotate(360deg);-ms-transform:rotate(360deg);transform:rotate(360deg);}}",
              "@keyframes iVXCSc{from{-webkit-transform:rotate(0deg);-ms-transform:rotate(0deg);transform:rotate(0deg);}to{-webkit-transform:rotate(360deg);-ms-transform:rotate(360deg);transform:rotate(360deg);}}",
            ],
            "toString": [Function],
          },
          " 2s linear infinite;
",
        ],
      },
      "displayName": "styled.div",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "div",
      "toString": [Function],
      "usesTheme": false,
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
